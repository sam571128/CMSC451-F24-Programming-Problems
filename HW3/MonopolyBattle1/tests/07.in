100 100 1000
88 12
27 16
85 22
81 7
8 98
33 43
58 66
6 96
45 42
94 76
27 16
15 51
39 64
74 95
43 20
44 10
41 5
80 91
83 25
47 8
75 7
88 79
67 63
41 88
89 99
32 21
36 63
70 22
69 53
19 41
99 45
88 100
20 83
20 26
74 36
59 69
7 29
90 19
46 64
69 20
95 57
80 38
82 14
47 1
89 75
14 36
94 63
33 74
78 37
37 53
84 99
57 7
70 91
70 88
3 85
76 93
76 58
51 56
59 37
98 37
15 85
99 6
46 24
93 33
26 95
49 29
52 67
66 72
52 65
13 48
19 95
9 37
15 77
100 13
15 25
28 49
80 11
63 65
5 38
100 11
80 21
23 89
94 99
41 74
63 59
9 61
76 77
33 34
16 59
19 83
80 33
100 93
98 13
16 24
42 72
44 67
60 87
99 81
31 83
22 100
