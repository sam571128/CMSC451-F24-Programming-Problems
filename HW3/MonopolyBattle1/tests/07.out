113 139 221 113 128 135 135 113 206 128 211 199 175 113 248 139 199 175 109 113 183 113 199 124 203 273 116 113 124 135 164 113 258 258 122 360 203 113 139 273 176 185 552 109 229 199 206 159 175 113 253 123 121 121 182 158 192 185 113 109 113 135 121 121 139 503 113 148 138 295 124 113 158 211 469 121 124 122 158 109 221 172 139 124 211 273 113 199 113 273 206 113 148 128 121 303 148 248 109 214 